internal/oc/release.go

```diff
@@ -343,19 +343,35 @@ func (r *release) Extract(log logrus.FieldLogger, releaseImage string, releaseIm
 	return path, err
 }
 
-func (r *release) GetReleaseBinaryPath(releaseImage string, cacheDir string) (workdir string, binary string, path string) {
-	workdir = filepath.Join(cacheDir, releaseImage)
+func (r *release) GetReleaseBinaryPath(releaseImage string, cacheDir string, ocpVersion string) (workdir string, binary string, path string, err error) {
```

Comment:
I wonder maybe we can use https://github.com/openshift/assisted-service/pull/6304/files#diff-3b8a020ef087b81b6b45d56bef6dd8d32fcf9d50dfe3d91fd369755b253cfff8R299 
in order to get release version?

Reply:
I thought about that, but this is called pretty frequently and I didn't want to be running `oc ...` every time